CIP-0021/README.md

```diff
@@ -70,7 +72,7 @@ Unless mentioned otherwise in this CIP, optional empty lists and maps must not b
 
 **Outputs**
 
-Outputs containing no multi-asset tokens must be serialized as a simple tuple, i.e. `[address, coin]` instead of `[address, [coin, {}]]`.
+Outputs containing no multi-asset tokens must be serialized as a simple tuple, i.e. `[address, coin, ?datum_hash]` instead of `[address, [coin, {}], ?datum_hash]`. In addition, including `datum_hash` is only allowed if the output `address` is a script address.
```

Comment:
```suggestion
Outputs containing no multi-asset tokens must be serialized as a simple tuple, i.e. `[address, coin, ?datum_hash]` instead of `[address, [coin, {}], ?datum_hash]`. In addition, including `datum_hash` is only allowed if the output `address` contains a script hash in its payment part.
```

Reply:
https://github.com/vacuumlabs/CIPs/pull/6/commits/62e1226251493954b56b1f51a950c83cb365608a